xee/ext_integration_test.py

```diff
@@ -349,7 +349,25 @@ def test_open_dataset__sanity_check(self):
     )
     for v in ds.values():
       self.assertIsNotNone(v.data)
-      self.assertFalse(v.isnull().all(), 'All values are null!')
```

Comment:
nit: we should add a test case for this as well. I mean pass scale as 25.0 & projection with -ve y_scale.

Reply:
Added.